fusion_docs/index.md

```diff
@@ -39,6 +39,13 @@ When pipelines run with cloud storage, tasks typically read data from a bucket,
 
 The result is significant overhead for every task. Fusion enables direct file access to cloud object storage, eliminating unnecessary I/O and dramatically reducing data movement and overall runtime.
 
+### Checkpoint and recovery terminated instances
+
+Running pipelines on AWS Spot instances can lead to interruptions, causing tasks to restart from the beginning and waste time and compute resources.
+
+Fusion Snapshots enable checkpointing of running Nextflow processes, allowing tasks to be resumed on a different machine if interrupted.
```

Comment:
Need to emphasise that this is mid-task, otherwise "resume" sounds like the normal Nextflow behaviour.

Reply:
Added "mid-task" to description.